evil-collection-reftex.el

```diff
@@ -0,0 +1,140 @@
+;;; evil-collection-reftex.el --- Bindings for `reftex'. -*- lexical-binding: t -*-
+
+;; Copyright (C) 2017 Pierre Neidhardt
+
+;; Author: Maximiliano Sandoval <msandova@protonmail.com>
+;; Maintainer: James Nguyen <james@jojojames.com>
+;; Pierre Neidhardt <ambrevar@gmail.com>
+;; URL: https://github.com/jojojames/evil-collection
+;; Version: 0.0.1
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: evil, reftex, tools
+
+;; This file is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published
+;; by the Free Software Foundation; either version 3, or (at your
+;; option) any later version.
+;;
+;; This file is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; For a full copy of the GNU General Public License
+;; see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Evil bindings for `reftex-mode'.
+
+;;; Code:
+(require 'evil)
+(require 'reftex-ref nil t)
+(require 'reftex-ref nil t)
+(require 'reftex-cite nil t)
+
+;; original code can be found in reftex-ref.el
+(defconst reftex-select-label-prompt
+  "Select: [RET]select [j]next [k]previous [gr]escan [go]context [q]uit [?]help"
+  )
+
+;; original code can be found in reftex-cite.el
+(defconst reftex-citation-prompt
+  "Select: [RET]select [j]next [k]previous [q]uit [?]help"
+  )
+
+;; original at reftex-ref.el
+(defconst reftex-select-label-help
```

Comment:
Nice! Are these created by you? You might want to prepend 'evil-collection-reftex' to these constants.

Reply:
reftex defines them to use as the prompt so prepending `evil-collection-` would break it, I don't know a clean way around. I don't have that much elisp experience, I will change them to `setq` for the moment.